src/vehicleFc.cpp

```diff
@@ -0,0 +1,52 @@
+#include "vehicle.h"
+
+#if (CURRENT_VEHICLE == FC)
```

Comment:
Each vehicle's cpp file is only compiled if that vehicle is selected.

Reply:
I added a comment in the settings file, but we can simplify this with an `ifdef` for the particular vehicle.

Reply:
Done!